tests/e2e/ctl_v3_auth_cluster_test.go

```diff
@@ -0,0 +1,137 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	"go.etcd.io/etcd/tests/v3/framework/config"
+	"go.etcd.io/etcd/tests/v3/framework/e2e"
+	"sync"
+	"testing"
+)
+
+const (
+	numKeys      = 10
+	writeThreads = 10
+
+	rootUser     = "root"
+	rootPassword = "root"
+	rootRole     = "root"
+	testUser     = "test"
+	testPassword = "testPassword"
+	testRole     = "testRole"
+)
+
+func TestAuthCluster(t *testing.T) {
+	e2e.BeforeTest(t)
+	cfg := e2e.NewConfigNoTLS()
+	cfg.ClusterSize = 1
+	cfg.SnapshotCount = 2
+	cfg.ClientAuthOpts = []config.ClientOption{e2e.WithAuth(rootUser, rootPassword)}
```

Comment:
Please remove this

Reply:
We still need to override client configuration for epc.StartNewProc() to work correctly.
Or did I misunderstand you?